docs/guides/Smart Contracts/1.index.md

```diff
@@ -38,3 +38,14 @@ namespace mycontract {
 }
 ```
 :::
+
+## Building a contract
+
+There are 3 options to build a contract at the moment:
+
+* Via [eosio-cpp](../Docker/docker-contract-development-flow.md) with docker
```

Comment:
Here you better link to guides/Docker/getting-started.md to not skip other steps plus it basically covers the whole build process